controllers/testdata/declarative/case_2/expected/created/mlpipelines-ui_deployment.yaml

```diff
@@ -46,6 +46,10 @@ spec:
               value: ds-pipeline-testdsp2
             - name: ML_PIPELINE_SERVICE_PORT
               value: '8888'
+            - name: METADATA_ENVOY_SERVICE_SERVICE_HOST
+              value: ds-pipeline-metadata-envoy-testdsp2
+            - name: METADATA_ENVOY_SERVICE_SERVICE_PORT
+              value: "9090"
```

Comment:
Can you also add a test case for ensuring mlmd is deployed when the field is enabled